src/components/Table/Table.test.tsx

```diff
@@ -0,0 +1,219 @@
+import { render, screen } from '@testing-library/react'
+import { DefaultCellRenderer } from 'components/Table/defaults'
+import {
+  DateRangeColumnFilter,
+  NumberRangeColumnFilter,
+} from 'components/Table/filters'
+import { Column } from 'react-table'
+import { DebugProvider } from 'test/testMocks'
+import { ObjectWithStringKeys } from 'types/custom-types'
+
+import { Table } from './Table'
+
+interface TestData extends ObjectWithStringKeys {
+  name: string
+  secondValue: string | JSX.Element | number
+}
+
+let tableColumns: Column<TestData>[], tableData: TestData[]
+
+describe('Table', () => {
+  beforeEach(() => {
+    tableColumns = [
+      {
+        Header: 'Name',
+        accessor: 'name',
+        minWidth: 120,
+        maxWidth: 180,
+        width: 130,
+      },
+      {
+        Header: 'Grade',
+        accessor: 'secondValue',
+        Cell: undefined,
```

Comment:
Looks like this line is causing the rest of the Table tests to fail. Deleting this would fix and allow the default cell renderer to be used.